index.js

```diff
@@ -9,6 +9,20 @@ var minimums = {
   aaaLarge: 4.5
 };
 
+// Adapted from axe-core
+// https://github.com/dequelabs/axe-core/blob/4ef74435ac92a88c602690ac1e243c404f750d00/lib/commons/color/contrast.js#L88-L96
+//
+// TODO: Attribute this properly.
+var flattenColors = function (color, bg) {
```

Comment:
Can replace this with `Color().mix`, it's just mixing the solid color with a percentage (inverse of the alpha level) of the transparent color.